src/vehicle.cpp

```diff
@@ -1296,6 +1309,98 @@ void vehicle::use_controls()
     refresh();
 }
 
+void vehicle::use_controls_simple()
```

Comment:
A separate function that implements part of the functionality of a bigger one is a pain to maintain.

You could split the `use_controls` section into a simple part and advanced part. Then you could just have the advanced part in an `if( not_simple )`. You wouldn't need to handle advanced options in a switch, just populating the menu.